tests/default_vars.sh

```diff
@@ -501,6 +501,7 @@ export DOCN_CDEPS=false
 export CDEPS_INLINE=false
 export POSTAPP='global'
 export USE_MERRA2=.true.
+export NESTED=.false.
```

Comment:
Do we want to use "DO_NEST" to be consistent with the global-workflow?

Reply:
No. We want `NESTED` so it is consistent with the namelist and model.

Reply:
Got it.